src/script/script.h

```diff
@@ -68,139 +68,148 @@ std::vector<unsigned char> ToByteVector(const T& in)
-    OP_PICK = 0x79,
-    OP_ROLL = 0x7a,
-    OP_ROT = 0x7b,
-    OP_SWAP = 0x7c,
-    OP_TUCK = 0x7d,
+    OP_TOALTSTACK = 0x6b,   // pop an item from the main stack onto the alt stack
+    OP_FROMALTSTACK = 0x6c, // pop an item from the alt stack onto the main stack
+    OP_2DROP = 0x6d,        // remove the two top stack items
+    OP_2DUP = 0x6e,         // duplicate top and second from top stack items
+    OP_3DUP = 0x6f,         // duplicate top, second from top and third from top stack items
+    OP_2OVER = 0x70,        // copy third and fourth from top stack items to the top
+    OP_2ROT = 0x71,         // move fifth and sixth from top stack items to the top
+    OP_2SWAP = 0x72,        // swap: top and second from top <-> third and fourth from top items of stack
+    OP_IFDUP = 0x73,        // if top stack value is false, duplicate it
+    OP_DEPTH = 0x74,        // push the current number of stack items onto the stack
+    OP_DROP = 0x75,         // remove top stack item
+    OP_DUP = 0x76,          // duplicate top stack item
+    OP_NIP = 0x77,          // remove second from top stack item
+    OP_OVER = 0x78,         // copy second from top stack item to the top
+    OP_PICK = 0x79,         // copy item N back in stack to the top
+    OP_ROLL = 0x7a,         // move item N back in stack to the top
+    OP_ROT = 0x7b,          // move third from top stack item to the top
+    OP_SWAP = 0x7c,         // swap top two items of stack
+    OP_TUCK = 0x7d,         // copy top stack item and insert before second from top item
 
     // splice ops
-    OP_CAT = 0x7e,
-    OP_SUBSTR = 0x7f,
-    OP_LEFT = 0x80,
-    OP_RIGHT = 0x81,
-    OP_SIZE = 0x82,
+    OP_CAT = 0x7e,    // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_SUBSTR = 0x7f, // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_LEFT = 0x80,   // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_RIGHT = 0x81,  // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_SIZE = 0x82,   // push the length of top stack item (not pop the top element whose size is inspected)
 
     // bit logic
-    OP_INVERT = 0x83,
-    OP_AND = 0x84,
-    OP_OR = 0x85,
-    OP_XOR = 0x86,
-    OP_EQUAL = 0x87,
-    OP_EQUALVERIFY = 0x88,
-    OP_RESERVED1 = 0x89,
-    OP_RESERVED2 = 0x8a,
+    OP_INVERT = 0x83,      // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_AND = 0x84,         // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_OR = 0x85,          // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_XOR = 0x86,         // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_EQUAL = 0x87,       // return 1 if the inputs are equal, 0 otherwise
```

Comment:
Should probably also mention here that the inputs are popped from the stack? E.g. something like
`pop two top stack items and push 1 if they are equal, 0 otherwise`